### ISSUE TITLE
Accessing Database on the main thread of minecraft

### ISSUE THREAD

**Issue by ghost** [Mineaurion/AurionVoteListener#25]
"it's the voting plugin (AurionVoteListener) timing out when trying to access the database on the main thread.
the database config might be wrong, or the database is acting weird, or something, but it's blocking the main thread for 30 seconds and so nothing can run during that time.
the plugin author should really be doing that stuff async"

"@NikkuMiru Even saving to file, it shouldn't be doing that on the main thread. Report bug to author."

**Comment by Yann151924** [comment]
The plugin is currently under rewrite, but where the conversation come from, i'm curious :) ?

**Comment by ghost** [comment]
From me and a other Guy in the Sponge Discord since i had a problem with it on my server. Cause when i wrote a command for example /sponge i get like 10 warn lines in console from the vote plugin and then i time out

**Comment by ghost** [comment]
after i reloaded the plugin

**Comment by Yann151924** [comment]
I currently make a full rewrite of the plugin so i think the prob will be solved when i finish.
Can you upload this log on pastebin, so i can take a look and make a quick fix if i can ?

**Comment by Yann151924** [comment]
Log :

```
12:10:43] [Server thread/INFO] [STDERR]: [com.mineaurion.aurionvotelistener.SQL.SwitchSQL:TotalsVote:56]: java.sql.SQLTransientConnectionException: HikariPool-2 - Connection is not available, request timed out after 30000ms.
[12:10:43] [Server thread/INFO] [STDERR]: [com.mineaurion.aurionvotelistener.SQL.SwitchSQL:TotalsVote:56]:  at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:601)
```

The prob come from when dbfile is set and reload is launched

**Comment by Yann151924** [comment]
Fix in the latest version 2.5.6
